docs/source/deployment/lambda.md

```diff
@@ -58,6 +59,226 @@ exports.graphqlHandler = server.createHandler();
 
 Finally, **pay close attention to the last line**. This creates an export named `graphqlHandler` with a Lambda function handler.
 
+## Deploying with the AWS Serverless Application Model (SAM)
+
+The [AWS Serverless Application Model (AWS SAM)](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html) is an open-source framework that you can use to build serverless applications on AWS.
+
+### Configuring your SAM template.yaml
+
+An AWS SAM template file closely follows the format of an AWS CloudFormation template file, which is described in [Template anatomy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-anatomy.html) in the AWS CloudFormation User Guide.
+
+For the sake of this example, the following file can just be copied and pasted into the root of your project.
+
+```
+# template.yaml
+
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Resources:
+  GraphQLFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: graphql/
+      Handler: app.graphqlHandler
+      Timeout: 5 #
+      MemorySize: 256
+      Runtime: nodejs14.x
+      Events:
+        AutomaticPersistedQueries:
```

Comment:
This is also an odd name.  Because GET on / also serves landing pages, and GET GraphQL requests don't have to use APQs, and APQs don't have to be via GET.  (Yes, I realize that useGETForHashedQueries exists in Apollo Client.)

I think these three events should just be called GraphQLPOST, GraphQLGET, GraphQLOPTIONS or something? Or maybe they can be one event?

Reply:
I changed the naming of these, unfortunately I couldn't figure out how to combine them all (I think they have to be listed separately)